src/modules/flight_mode_manager/tasks/ManualAltitude/FlightTaskManualAltitude.hpp

```diff
@@ -88,7 +89,9 @@ class FlightTaskManualAltitude : public FlightTask
 					(ParamFloat<px4::params::MPC_LAND_SPEED>)
 					_param_mpc_land_speed, /**< desired downwards speed when approaching the ground */
 					(ParamFloat<px4::params::MPC_TKO_SPEED>)
-					_param_mpc_tko_speed /**< desired upwards speed when still close to the ground */
+					_param_mpc_tko_speed, /**< desired upwards speed when still close to the ground */
+					(ParamFloat<px4::params::MPC_VEL_MANUAL>) _param_mpc_vel_manual,
+					(ParamFloat<px4::params::MPC_ACC_HOR>) _param_mpc_acc_hor
```

Comment:
I don't see where you used `_param_mpc_vel_manual` and `_param_mpc_acc_hor` in `FlightTaskManualAltitude.hpp`

Reply:
used them in FlightTaskManualAltitude.cpp

Reply:
where? I only see them used in [FlightTaskManualAcceleration.cpp](https://github.com/PX4/PX4-Autopilot/pull/23667/files#diff-8d0af4d9e531c4e5b5042aa541daa75f8c8375bfd4467096c2f186a8a501a549)